.github/workflows/ci.yml

```diff
@@ -56,22 +81,38 @@ jobs:
           arch: arm64
 
     runs-on: ${{ matrix.host }}
-    needs: build
+    needs: [build, build-32bit]
     steps:
     - name: install rust
+      if: ${{ matrix.target.arch == 'x64'}}
       uses: hecrj/setup-rust-action@v1
       with:
         rust-version: stable
+    
+    - name: install rust i686
+      if: ${{ matrix.target.arch == 'x86' }}
+      uses: hecrj/setup-rust-action@v1
+      with:
+        rust-version: stable
+        targets: "i686-unknown-linux-gnu"
```

Comment:
Shouldn't be necessary. xbuild automatically downloads the required targets using rustup

Reply:
Maybe it is supposed to, but 
```
error[E0463]: can't find crate for `core`
  |
  = note: the `i686-unknown-linux-gnu` target may not be installed
  = help: consider downloading the target with `rustup target add i686-unknown-linux-gnu`
```
says otherwise.

Reply:
Then maybe something is missing somewhere...

Reply:
This used to be hardcoded but should be fixed now since https://github.com/rust-mobile/xbuild/pull/139. @iamwacko can you rebase, remove this setup from the CI (because `xbuild` must do it to provide a more-or-less out-of-the-box experience), and retest?